mopidy/internal/path.py

```diff
@@ -196,23 +196,23 @@ def find_mtimes(root, follow=False):
     return mtimes, errors
 
 
-def check_file_path_is_inside_base_dir(file_path, base_path):
-    assert not file_path.endswith(os.sep), (
-        'File path %s cannot end with a path separator' % file_path)
-
+def is_local_path_inside_base_dir(local_path, base_path):
```

Comment:
I don't think the `local_` prefix adds anything here. It just confuses a bit by indicating a link with the local extension.